src/pm/tui/polling.py

```
@@ -0,0 +1,388 @@
+            )
+            self._update_rate_limit(resp)
+            if resp.status_code == 200:
+                state = resp.json().get("state", "pending")
+                if state == "success":
+                    pr.ci_status = "passing"
+                elif state == "failure" or state == "error":
+                    pr.ci_status = "failing"
+                else:
+                    pr.ci_status = "pending"
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)